### ISSUE TITLE
HDR videos are washed out

### ISSUE THREAD

**Issue by eblikstad** [tonyfederer/EasyVideoScreensaver#11]
Hi

I have downloaded a video from Youtube to test the screensaver.

- https://www.youtube.com/watch?v=LXb3EKWsInQ&t=224s

When viewing the downloaded video (MP4) in Windows everything seems fine however when I configure the screensaver to use this video and test it, the video looks washed out. I have tested 2 different videos downloaded from Youtube. I haven't enabled HDR in Windows.

Regards,
Espen Blikstad

**Comment by brad992** [comment]
Yes I am experiencing the same thing. I have an OLED screen and my blacks are very washed out. Would extremely appreciate some sort of solution to get my blacks black again. It must not be activating HDR.

I'm using Windows 11 and LG OLED C3

**Comment by tifandev** [comment]
I know I may be years later but I found a fix using GPT. Basically you have to force your GPU to handle colors:

If you have an NVIDIA GPU:

Open NVIDIA Control Panel → Adjust video color settings.
Under "How do you make color adjustments?", select "With NVIDIA settings".
Set Dynamic Range to Full (0-255).
Apply changes and restart the screensaver.

For AMD GPUs:

Open Radeon Settings → Display → Pixel Format.
Set it to Full RGB (0-255) instead of Limited.

**Comment by brad992** [comment]
> I know I may be years later but I found a fix using GPT. Basically you have to force your GPU to handle colors:
> 
> If you have an NVIDIA GPU:
> 
> Open NVIDIA Control Panel → Adjust video color settings. Under "How do you make color adjustments?", select "With NVIDIA settings". Set Dynamic Range to Full (0-255). Apply changes and restart the screensaver.
> 
> For AMD GPUs:
> 
> Open Radeon Settings → Display → Pixel Format. Set it to Full RGB (0-255) instead of Limited.

omggg thank you so much! I have NVIDIA and that worked for me. I just hope this doesn't cause any negative effects during normal usage, haven't noticed any difference yet. 

Anyways, I'm gonna be very much enjoying my new screensaver now!! For anyone that cares, I'm made my own screensaver using loops from this guys website: https://bleuje.com/animationsite/. They are the perfect vibe
